src/System.Text.Primitives/System/Text/Encoding/Utf16/Utf16TextEncodingLE.cs

```diff
@@ -7,113 +7,145 @@ namespace System.Text.Utf16
 {
     class Utf16TextEncodingLE : TextEncoder
     {
-        public override bool TryEncodeFromUtf8(ReadOnlySpan<byte> utf8, Span<byte> buffer, out int bytesWritten)
+        #region Encoding Constants
+
+        // TODO: Some of these members are needed only in Utf16LittleEndianEncoder.
+        //       Should we add the usage of them to UnicodeCodePoint class and merge this class with it?
+        private const uint Utf16HighSurrogateFirstCodePoint = 0xD800;
+        private const uint Utf16HighSurrogateLastCodePoint = 0xDFFF;
```

Comment:
Those constants are now removed. I meant to have them on just the static class, but forgot to remove them when I split the classes.